cmd/api/src/model/sso_provider.go

```diff
@@ -30,6 +30,16 @@ type SSOProvider struct {
 	Serial
 }
 
+// AuditData returns the fields to log in the audit log
+func (s SSOProvider) AuditData() AuditData {
+	return AuditData{
```

Comment:
You might be able to get fancy here and based on the type add in the details tied to the child provider info with a switch case like on our LIST endpoint 🤔 

You'll need to do the preload trick that @ddlees mentioned `db.Preload("OIDCProvider").Preload("SAMLProvider")`